crates/bevy_window/src/window.rs

```diff
@@ -963,3 +975,26 @@ pub enum WindowTheme {
     /// Use the dark variant.
     Dark,
 }
+
+#[derive(Debug, Copy, Clone, PartialEq, Reflect)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
+#[reflect(Debug, PartialEq, Default)]
+pub struct EnabledButtons {
```

Comment:
There's not doc here :(

I think the doc in `Window` should go straight to the point, and the doc for this struct should go more into details on the specificities on each platform, interaction with `decoration` and `resizable`, detail every field even if it's kinda obvious...

Reply:
I changed the docstrings, can you check whether the new ones are any good? I am by no means a documentation expert so feel free to modify anything you don't like, even if it's a misplaced dot. 😄

Reply:
Looks good to me ^^